shared/chat/conversation/team-mention/index.js

```diff
@@ -0,0 +1,117 @@
+// @flow
+import React from 'react'
+import Text from '../../../common-adapters/text'
+import {Box2} from '../../../common-adapters/box'
+import * as Styles from '../../../styles'
+import TeamInfo from '../../../profile/user/teams/teaminfo'
```

Comment:
we now do 
```
const Kb = {Box2, Text}
```
in files that can't import * common so the jsx looks the same (`Kb.Box2`) etc